### PR TITLE
Docstring

### PR DESCRIPTION
**Pull request by albrja**
### Description

- *Category*: 
- *JIRA issue*: https://jira.ihme.washington.edu/browse/MIC-XYZ

### CODE REVIEW

File: src/vivarium/component.py [hunk 1]
```diff
@@ -579,7 +579,14 @@ def get_initialization_parameters(self) -> Dict[str, Any]:
         }
 
     def create_lookup_tables(self, builder: "Builder") -> None:
-
+        """
+        Method to create standard lookup tables for the component. This will create a
+        lookup table for each lookup key in self.standard_lookup_tables property. If
+        there is desire to add additional lookup tables, users can override this method
+        by calling the super method and adding additional lookup tables or overriding the
+        standardary lookup tables property if they want to add additional lookup tables
```
Comment (patricktnast) [inline src/vivarium/component.py:587]: `standardary` typo?

File: src/vivarium/component.py [hunk 2]
```diff
@@ -579,7 +579,14 @@ def get_initialization_parameters(self) -> Dict[str, Any]:
...
+        that can be build in the standard way (shown below).
```
Comment (stevebachmeier) [inline src/vivarium/component.py:588]: typo "built" with a "t"

File: src/vivarium/component.py [hunk 3]
```diff
@@ -579,7 +579,14 @@ def get_initialization_parameters(self) -> Dict[str, Any]:
         }
 
     def create_lookup_tables(self, builder: "Builder") -> None:
-
+        """
+        Method to create standard lookup tables for the component. This will create a
+        lookup table for each lookup key in self.standard_lookup_tables property. If
+        there is desire to add additional lookup tables, users can override this method
+        by calling the super method and adding additional lookup tables or overriding the
```
Comment (stevebachmeier) [inline src/vivarium/component.py:586]: This last sentence is difficult to read b/c of lots of "lookup tables". Maybe:
`If there is desired to add additional lookup tables, users have two options: (1) override this method by calling the super method and adding them, or (2) overriding the standard 'lookup_tables' property`

Or something like that.

Comment (albrja) [reply src/vivarium/component.py:586]: I like this.

File: src/vivarium/component.py [hunk 4]
```diff
@@ -579,7 +579,13 @@ def get_initialization_parameters(self) -> Dict[str, Any]:
         }
 
     def create_lookup_tables(self, builder: "Builder") -> None:
-
+        """
+        Method to create standard lookup tables for the component. This will create a
+        lookup table for each lookup key in self.standard_lookup_tables property. If
+        there is desired to add additional lookup tables, users have two options:
```
Comment (stevebachmeier) [inline src/vivarium/component.py:585]: typo (from my suggestion - oops): "there is desired"

**Pull request merged**